using-mobile-extensions/mobile-core/identity/identity-event-reference.md

```diff
@@ -4,8 +4,160 @@
 
 The following events are handled by the Identity extension:
 
+### Analytics Identity Response
+
+#### Event Details
+
+| Type                          | Source                                 | Paired | Direction                                                    |
+| :---------------------------- | :------------------------------------- | :----- | :----------------------------------------------------------- |
+| com.adobe.eventType.analytics | com.adobe.eventSource.responseIdentity | True   | This is the end of the paired [Event: v5 Analytics Request Identity](https://wiki.corp.adobe.com/display/ADMSMobile/Event%3A+v5+Analytics+Request+Identity) event containing the AID value |
+
+#### Event Description
+
+This event is a response from the Analytics module that contains the unique tracking identifier. This value may be null if identity module is enabled and there is a valid MID generated.
+
+#### Generators
+
+This event will be generated by:
+
+- calling the Analytics module upon processing the com.adobe.eventSource.requestIdentity event.
+
+#### Event Data Payload Definition
+
+Definition of the key/value pairs that will be present in this event
+
+| Key  | Value Type | Optional | Description                             |
+| :--- | :--------- | :------- | :-------------------------------------- |
+| aid  | String     | No       | The analytics tracking identifier value |
+
+#### Event Data Example
+
+**sample**
+
+```
+`{``    ``"aid" : "aidValue"``}`
+```
+
+
+### Audience Manager Content Response
+
+#### Event Description
+
+This event has 2 purposes:
+
+1. Deliver the Audience Manager profile to the original requester
+2. Deliver the Audience Manager profile to any module that may be listening for updates.
+
+This event will be generated in the following scenarios:
+
+- When a request is made to update the Audience manager profile 
+- When a request is made to retrieve the current Audience manager profile 
+
+#### Event Details
+
+| Type                                | Source                                | Paired | Direction                                                    |
+| :---------------------------------- | :------------------------------------ | :----- | :----------------------------------------------------------- |
+| com.adobe.eventType.audienceManager | com.adobe.eventSource.responseContent | True   | This is the response to a paired event. This should be published both with and without a paired ID |
```

Comment:
We can remove the extra detail about paired id as for third party developers there is no concept of pairId. 
You can include a link to the Audience response content event https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-audience-manager/audience-manager-event-reference#audience-manager-content-response-1